wp/plugin/wp.ml

```diff
@@ -126,14 +106,10 @@ let compare_projs (proj : project) (file1: string) (file2 : string)
   let main_sub1 = find_func_err subs1 func in
   let main_sub2 = find_func_err subs2 func in
   let env1, env2 =
-    if inline || has_inline_funcs to_inline then
-      let to_inline1 = find_inline_funcs to_inline ~to_find:subs1 ~to_check:subs2 in
-      let to_inline2 = find_inline_funcs to_inline ~to_find:subs2 ~to_check:subs1 in
-      Pre.mk_inline_env ctx var_gen ~subs:subs1 ~arch:arch ~to_inline:to_inline1,
-      Pre.mk_inline_env ctx var_gen ~subs:subs2 ~arch:arch ~to_inline:to_inline2
-    else
-      Pre.mk_default_env ctx var_gen ~subs:subs1 ~arch:arch,
-      Pre.mk_default_env ctx var_gen ~subs:subs2 ~arch:arch
+    let to_inline1 = match_inline to_inline subs1 in 
+    let to_inline2 = match_inline to_inline subs2 in 
```

Comment:
Do you think it's a good idea to have an info log about which functions are going to be inlined?

Reply:
Smart.